### DISCUSSION TITLE
GitHub build with Jekyll failed   Invalid Date: '{}' is not a valid datetime.

### DISCUSSION THREAD

**Discussion by tianhuahao** [hexojs/hexo#5588] [Question / Answer] [answered]
![image](https://github.com/user-attachments/assets/9f4530e5-fa22-4f51-b785-10c2c42c6768)

But this folder is not included in my project

![image](https://github.com/user-attachments/assets/00ef61e4-4a39-437d-9f78-0f2544b15115)

**Comment by stevenjoezhang** [comment, marked as answer]
Hexo is irrelevant with Jekyll and you should not build your site with it. Try creating a file named `.nojekyll` in the root of your pages repo.
